db/badgerdb/badger_test.go

```diff
@@ -1,124 +1,57 @@
 package badgerdb
 
 import (
-	"bytes"
-	"strconv"
+	"encoding/binary"
 	"testing"
 
+	qt "github.com/frankban/quicktest"
 	"go.vocdoni.io/dvote/db"
+	"go.vocdoni.io/dvote/db/dbtest"
```

Comment:
move this package to `db/internal/dbtest`. we don't want it to be importable by others.